crates/autopilot/src/decoded_settlement.rs

```diff
@@ -635,89 +645,103 @@ mod tests {
     #[test]
     fn total_fees_test() {
         // transaction hash:
-        // 0x4ed25533ae840fa36951c670b1535265977491b8c4db38d6fe3b2cffe3dad298
+        // 0x8f39bb793d3beac9aa944c5cc23e3e8677f639bdf87c2df9eb869a2875a8df7a
```

Comment:
Changed the order to test, since tx `0x4ed25533ae840fa36951c670b1535265977491b8c4db38d6fe3b2cffe3dad298` contains old order for which the subsidies were applied and therefore the new fee calculation does not apply.